src/components/MapContainer.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Map from './Map'
+import { connect } from 'react-redux'
+
+class MapContainer extends React.Component {
+	render() {
+		return (
+			<Map className="map-container" config={this.props.config}/> 
```

Comment:
OK I think I know what is happening here. The `className` needs to be passed through props until it lands on a DOM element. What we should probably do is have a `<div>` wrapper around `<Map>` to hold the `className` instead of passing it to `Map`.

Also - I just noticed you're using tabs here, let's stick to spaces for this project.